### PR TITLE
Added italian translation to readme for the Incident listener Plugin

### PR DESCRIPTION
**Pull request by enisinanaj**

### CODE REVIEW

File: README-it.md [hunk 1]
```diff
@@ -0,0 +1,467 @@
...
+1. apri [subprocess_example_parent.bpmn](https://github.com/jit-open/incident-listener-spring-boot-demo/blob/master/src/main/resources/subprocess_example_parent.bpmn),
+2. nell'attività `Set up incident listener` cambia la variabile `incidentNotificationReceiver` e `incidentNotificationCc` in indirizzi mail effettivi,
+3. avvia l'applicazione, e
+4. avvia il processo `Subprocess Example (Parent)` nella task list.
+
+## Versioni
+
+ * `1.0.0`: Initial version.
+
+## Maintainer
```
Comment (dptij) [inline README-it.md:441]: Please remove the `Maintainer` section. We are offering this plugin "as-is", i. e. without support.

File: README-it.md [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+
+TODO
+
+## Contributors
+
+TODO
```
Comment (dptij) [inline README-it.md:447]: Please add here

```
 * Dmitrii Pisarenko
 * Miguel De los Santos La Paz
```

Comment (enisinanaj) [reply README-it.md]: done

File: README-it.md [hunk 3]
```diff
@@ -0,0 +1,464 @@
+# Plugin per notifiche via Email in caso di problemi
```
Comment (dptij) [inline README-it.md:1]: Please add a link in file `README.md` to `README-it.md` as shown below.

`README.md` before change:

```
This page is available in English (this document), [German](README-de.md), [Russian](README-ru.md), and [Slovenian](README-si.md).
```

`README.md` after change:

```
This page is available in English (this document), [German](README-de.md), [Russian](README-ru.md), [Slovenian](README-si.md), and [Italian](README-it.md).
```

**Review by bassgelo — APPROVED** [review]

**Pull request merged**